README.md

```diff
@@ -84,10 +84,41 @@ networksetup -listallhardwareports
 
 *The name of a network interface usually looks similar to this: `en0`.*
 
+### Windows
+
+Run a Python shell on your system (you should be able to do so by running python or python3 without any parameters) and then typing:
+
+```
+import netifaces
+netifaces.interfaces()
+```
+
+*The name of a network interface usually looks similar to this: `{00000000-0000-0000-0000-000000000000}`.*
+
+To find the correct interface, use the following command in a python shell, repeating it using each network interface name until a matching IP address for the Homebridge host system is identified:
+
+```
+netifaces.ifaddresses('{interface-name-here}')
+```
+
+In order for the plugin to execute properly within Homebridge and retrieve the input names and TV status, you must also change the Local System Account associated with the Homebridge service using the following steps:
+
+```
+Press the Windows Key + R
+services.msc
+Find the Homebridge Service and double-click
+Switch to the "Log On" tab
+Change "Local System Account" to "This Account" and enter your user name (usually .\username)
+-OR- 
+click "Browse..." and search for your username
+Enter your login password in the Password fields
+Press "Apply" and "OK" and restart the service if prompted
+```
```

Comment:
No need to make this monospaced, IMO.

```suggestion
- Press the Windows Key + R.
- Type `services.msc`.
- Find the Homebridge Service and double-click.
- Switch to the "Log On" tab.
- Change "Local System Account" to "This Account" and enter your user name (usually .\username).

**or**

- Click "Browse..." and search for your username.
- Enter your login password in the Password fields.
- Press "Apply" and "OK" and restart the service if prompted.
```

Reply:
Ah ok! Yeah, then maybe going on one line is better: I updated my suggestion, please let me know what you think.

Reply:
Looks good!